<launch>

	<!-- Joy node -->
	<node respawn="true" pkg="joy" type="joy" name="joy" >
		<param name="dev" type="string" value="/dev/input/js0" />
	</node>

	<!-- Axes and scaling -->
	<param name="axis_linear" value="1" type="int"/>
	<param name="axis_angular" value="0" type="int"/>
	<param name="scale_linear" value="0.20" type="double"/>
	<param name="scale_angular" value="0.8" type="double"/>
	
	<!-- At last - teleoperation node -->
	<node pkg="simple_apps" type="elektron_piotrek_joy" name="telepiotrek" output="screen" />
<!--	<node pkg="simple_apps" type="hoover_node" name="hoovernode" output="screen" />	-->

	<include file="$(find elektron_bringup)/launch/elektron_base.launch" />
<!--	<include file="$(find elektron_bringup)/launch/elektron_sensors.launch" />		-->

<!--	<include file="$(find elektron_navigation)/launch/local_sick.launch" />	-->
<!--	<include file="$(find sensors_processing)/launch/laser_to_odom.launch" /		-->


</launch>
